### DISCUSSION TITLE
OCI Helm chart registry for Istio

### DISCUSSION THREAD

**Discussion by wkbrd** [istio/istio#52292] [Q&A] [answered]
Hi, my organization has standardized on OCI Helm chart registries.  The instructions to install Istio using Helm use a registry that does not appear to be OCI-based.  Is there an OCI registry where the Istio Helm charts can be downloaded?  If not, could support for one be added?

**Comment by howardjohn** [comment, marked as answer]
`gcr.io/istio-release/charts`. Would be great to get this documented !

**Reply by wkbrd** [reply]
Hi @howardjohn , according to this page, https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr, the GCR is going away.  Does that mean that Istio needs to make changes to accommodate this?

**Reply by howardjohn** [reply]
this change will not impact users, the images will remain at gcr.io, just backed by a different server.

**Reply by AntonOfTheWoods** [reply]
@howardjohn , am I correct in understanding this is still not documented?

**Reply by Red-GV** [reply]
Based on the website, this is still undocumented as all the references I have seen only refer to the Helm repository and make no mention of the OCI even existing. I didn't know this existed until I started looking at the Git issues.

**Reply by AntonOfTheWoods** [reply]
At the end of the day, one of us could also add the documentation...

**Comment by wkbrd** [comment]
Amazing! Thanks for sharing, @howardjohn  :-)
